package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slashauth/slashauth-react",
-  "version": "0.9.8",
+  "version": "0.10.8",
```

Comment:
Let's just make this a patch version.

If you do decide to update the minor version, reset patch version to 0 (<MAJOR>.<MINOR>.<PATCH>)